sklearn/kernel_approximation.py

```diff
@@ -245,9 +245,8 @@ class AdditiveChi2Sampler(BaseEstimator, TransformerMixin):
     References
     ----------
     See `"Efficient additive kernels via explicit feature maps"
-    <http://eprints.pascal-network.org/archive/00006964/01/vedaldi10.pdf>`_
+    <http://citeseer.ist.psu.edu/viewdoc/summary?doi=10.1.1.167.7024>`_
```

Comment:
We could link straight to the [PDF on the lead author's website](http://www.robots.ox.ac.uk/~vedaldi/assets/pubs/vedaldi10efficient.pdf).

It seems there is also an extended [journal version](http://www.robots.ox.ac.uk/~vedaldi/assets/pubs/vedaldi11efficient.pdf) that we might want to refer to instead. (The citation for this one would be `A. Vedaldi and A. Zisserman, Pattern Analysis and Machine Intellingence, 2011`).

Thoughts?